webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/page/webanno.js

```diff
@@ -39,3 +39,11 @@ $(document)
       }
     });
 
+//wrap given script in try-catch block
+function tryCatch(jsCall){
+	try{
+		jsCall();
+	}catch (e){
+		console.warn('Call terminated due to: ' + e + ' script was:\n' + jsCall + '\n', e.stack);
```

Comment:
I think trying to print "jsCall" won't print the script but rather some representation of the function. Did you test this?